### DISCUSSION TITLE
bi-weekly chosen but weekly shown?

### DISCUSSION THREAD

**Discussion by sartyx** [mampfes/hacs_waste_collection_schedule#2992] [Q&A]
Hi,

first of all if I get that right, a lot of tutorials online mention the usage of ics calendar files.
This is obsolete in the current version now or am I wrong? 

I installed it picked my city/street in the gui and configured for 14 day paper/plastic/bio/waste.
However the card shows this weekly stuff:

![grafik](https://github.com/user-attachments/assets/41cc113f-9867-4174-b4c4-1cf569e44d66)

thx

**Comment by 5ila5** [comment]
ICS calendar files are still used for a lot of (especially german/austrian) service providers (see the [list here](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/source/ics.md)) if you choose one of them in the GUI you will be asked to put in a ICS file URL. But there are a lot of other source that use some other way to get the dates (ICS parsing for files without a static source, from a HTML site, from a JSON API result, ....).


Could you please tell me which source you're using (in brackets behind the city when selecting your area) and tell me what you mean with `configured for 14 day paper/plastic/bio/waste` (share screenshots or tell where you entered what)

**Comment by sartyx** [comment]
Hi, it is the mymuell app, mymuell.de

these settings:
![grafik](https://github.com/user-attachments/assets/c5789b03-e646-4253-a71b-1b445410fde3)

There are options for weekly and bi-weekly. However turns out I get weekly when chosen bi-weekly.

**Comment by 5ila5** [comment]
That's not really what this option does. You can select types you want to modify. When you select your types there and hit submit (Speichern) you should get a new screen for every type you selected where you can modify this type. You probably want to select all types you DON'T want to show up and uncheck the `show` option.

This modifies the Customize argument you can find more detailed documentation for the YAML based configuration (which is 1:1 mapped to the GUI)  [here](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/installation.md#attributes-for-customize)
